addons/userconfig/cba_settings.sqf

```diff
@@ -16,18 +16,31 @@ force ace_finger_enabled = true;
 
 force ace_gforces_enabledFor = 0;
 
-force ace_goggles_effects = 0;
-
 force ace_hearing_enableCombatDeafness = false;
 
 force ace_map_mapShowCursorCoordinates = true;
 
-force ace_medical_ai_enabledFor = 0;                      // Medical system not enabled for AI
-force ace_medical_feedback_painEffectType = 2;            // Pain effect type is "Chromatic Abberation".
```

Comment:
Add back key

Reply:
there have been reports of chromatic aberration acting up, 
for now we should leave this undefined and let players decide what to use

Reply:
Either we have "Chromatic Abberation" or we use the other one as default

Reply:
Let's try the new "Pulsing Blur"